package.json

```diff
@@ -185,7 +185,7 @@
     "lodash": "^4.17.21",
     "luxon": "^2.0.2",
     "memoize-one": "^4.0.2",
-    "monaco-editor": "^0.19.2",
+    "monaco-editor": "^0.21.0",
```

Comment:
Needed a slightly new version so I could use this fix: https://github.com/microsoft/vscode/commit/3e500bb935ff3f5254a40aeca511ba0f71fd6b01